app/javascript/components/Places/Place/Header/index.js

```diff
@@ -6,11 +6,18 @@ import PlaceIcon from 'icons/location.svg'
 import Navbar from './Navbar'
 import styles from './styles.module.scss'
 
+const Cover = ({ place }) => {
+  return (
+    <div className={styles.coverContainer}>
+      <div className={styles.cover} style={{ backgroundImage: `url(${place.cover})` }} />
+    </div>
+  )
+}
+
```

Comment:
I suggest to extract it to separate file and write `Cover` and `Header` in typescript.

Reply:
No need to extract `cover`, just use fragment `<>...</>`

Reply:
Rewrite it on typescript still valid suggestion